### DISCUSSION TITLE
500 Error after installing Data Importer

### DISCUSSION THREAD

**Discussion by kara-hyyyper** [pimcore/pimcore#16170] [Q&A / Support]
I am using Pimcore 11 and have been using the instructions listed here:
https://pimcore.com/docs/platform/Data_Importer/Installation

After installation when I try to open Pimcore its just a 500 error. I cannot access my PIM at all. 

I have added the required data to config/bundles.php and added the data importer data at the bottom of the list AFTER the "PimcoreDataHubBundle". So I believe the priority should not be a problem. 

I'm not getting any error messages in the logs regarding the Data Importer. But I do keep getting this error over and over again:
```
console.CRITICAL: Error thrown while running command "pimcore:maintenance --async --env=prod". Message: "The "--async" option does not exist."The "--async" option does not exist." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): The \"--async\" option does not exist. at /opt/bitnami/pimcore/vendor/symfony/console/Input/ArgvInput.php:223)","command":"pimcore:maintenance --async --env=prod","message":"The \"--async\" option does not exist."} []
```
When I run `bin/console pimcore:bundle:list` this is my result:
```
+------------------------------------------------------+---------+-----------+-------------+---------------+----------+
| Bundle                                               | Enabled | Installed | Installable | Uninstallable | Priority |
+------------------------------------------------------+---------+-----------+-------------+---------------+----------+
| PimcoreCustomerManagementFrameworkBundle             | ✔       | ✔         | ❌          | ✔             | 0        |
| OutputDataConfigToolkitBundle                        | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreAdminBundle                                   | ✔       | ✔         | ❌          | ✔             | 60       |
| PimcoreApplicationLoggerBundle                       | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreCustomReportsBundle                           | ✔       | ✔         | ❌          | ✔             | 20       |
| PimcoreDataHubBundle                                 | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreDataImporterBundle                            | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreEcommerceFrameworkBundle                      | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcorePaymentProviderPayPalSmartPaymentButtonBundle | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreGlossaryBundle                                | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreGoogleMarketingBundle                         | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreNewsletterBundle                              | ✔       | ✔         | ❌          | ✔             | 0        |
| NumberSequenceGeneratorBundle                        | ❌      | ❌        | ❌          | ❌            | 0        |
| ObjectMergerBundle                                   | ✔       | ✔         | ❌          | ❌            | 0        |
| PimcorePersonalizationBundle                         | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreSeoBundle                                     | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreSimpleBackendSearchBundle                     | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreStaticRoutesBundle                            | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreTinymceBundle                                 | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreUuidBundle                                    | ❌      | ❌        | ❌          | ❌            | 0        |
| PimcoreWebToPrintBundle                              | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreWordExportBundle                              | ✔       | ✔         | ❌          | ✔             | 0        |
| PimcoreXliffBundle                                   | ✔       | ✔         | ❌          | ✔             | 0        |
| Web2PrintToolsBundle                                 | ✔       | ✔         | ❌          | ✔             | 0        |
+------------------------------------------------------+---------+-----------+-------------+---------------+----------+
```

Any ideas what's wrong?

**Comment by kara-hyyyper** [comment]
The only way to get the 500 error to go away is by uninstalling the Data Importer Bundle with the `composer remove pimcore/data-importer` command and removing the Data Importer info from the config/bundles.php file. 

But I need the Data Importer to import data, so I would like to be able to install it.

**Comment by dvesh3** [comment]
you need to adjust your maintenance job to not use `--async` option as it is removed on v11. please see https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Upgrade_Notes/#maintenance-

**Reply by kara-hyyyper** [reply]
Do you know how I should go about doing this? I've recently installed pimcore with bitnami like a week ago, so Im surprised it didnt download completely updated. 

I followed this guide for pimcore updates: https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/
And I removed the cache as recommended here: https://github.com/pimcore/pimcore/issues/16161

I'm still getting the `Error thrown while running command "pimcore:maintenance --async --env=prod"` error message.

**Reply by kara-hyyyper** [reply]
![image](https://github.com/pimcore/pimcore/assets/125572770/91d88068-f6aa-4e31-bdc5-00ccbd9ee5cb)
This is where the word "async" shows up in the code editor. Am I supposed to change it in one of these files? And if I change it, what would I change it to?

**Reply by dvesh3** [reply]
This is related to maintenance cron job not packages https://pimcore.com/docs/pimcore/10.6/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job. Sorry, i have no experience with Bitnami setup so not sure where you need to update it. Maybe this will help https://docs.bitnami.com/general/apps/pimcore/configuration/configure-scheduler/

**Comment by kara-hyyyper** [comment]
Update: To remove the --async error messaging I just had to back up further in the file location on the SSH. I was too far deep so I couldn't see where the --async function was being called. I have resolved this issue, but I am still unable to install the Data Importer.
